compliance_checker/acdd.py

```diff
@@ -69,7 +69,7 @@ def check_recommended(self, ds):
     def check_suggested(self, ds):
         return [
             'contributor_name',
-            ('contributor_role', ['principalInvestigator', 'author']),
+            'contributor_role',
```

Comment:
It was changed, not sure when. There was a mailing list thing and @rsignell-usgs was on it and may have driven the change. Basically the discussion was about how everyone used it differently and it just needed to be comma separated.